Project_Mars/Project_Mars/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -1,29 +1,38 @@
-using SpecFlow_Mars.Pages;
-using SpecFlow_Mars.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+using System;
 
 
-namespace SpecFlow_Mars.StepDefinitions
+namespace Project_Mars.StepDefinitions
 {
     [Binding]
-    public class LanguageStepDefinitions : CommonDriver
+    [Parallelizable]
+    public class LanguageStepDefinitions 
     {
         LoginPage LoginPageobj = new LoginPage();
         HomePage HomePageobj = new HomePage();
         LanguageTab LanguageTabobj = new LanguageTab();
 
+        private IWebDriver driver;
 
+        public LanguageStepDefinitions(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
 
         [Given(@"User logs-in to Mars portal")]
         public void GivenUserLogs_InToMarsPortal()
         {
-            LoginPageobj.LogInActions();
-            HomePageobj.SelectLanguageTab();
+            LoginPageobj.LogInActions(driver);
```

Comment:
also please do not pass the driver around like, it is better create the driver in a common place and than to extend classes which need the driver to common driver

Reply:
Class extended to common driver